app/models.py

```diff
@@ -50,7 +47,7 @@ def __init__(self, username, first, last, email, phone):
         self.is_admin = False
 
     def __repr__(self):
-        return "<Contact '%s'>" % self.first_name + " " + self.last_name
+        return "<Contact '%s'>" % self.first_name + ' ' + self.last_name
```

Comment:
its kinda weird using both `%s` notation and `+ ' ' +`. Could you move the name combination to the line above and use `%s` for both?